source/_posts/2018-10-26-Going-to-ServerSide-swift-2018.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Going to ServerSide.swift 2018
+date: 2018-10-26 18:00:00
+tags: vapor,swift,linux,swift4,vapor3,serversideswift,backend
+authorIds:
+- stso
+- re
+- sikk
+categories:
+- Vapor
+---
+
+## The first conference in the world to focus on server-side Swift
+We (the Vapor team at Nodes) were excited when we heard that a conference focusing on server-side Swift got announced. We did get the news a bit early since the conference is run by [Tim Condon](https://twitter.com/0xtim), [Martin Lasek](https://twitter.com/martinlasek) and [Steffen D. Sommer](https://twitter.com/steffendsommer) where Martin and Steffen both works at Nodes. Before this conference, we would look at conferences focused around Swift or more traditional "backend" conferences which most likely would focus on another language than Swift. Being able to go to a conference that mixes backend together with our programming language of choice is ideal to us.
+
+The whole Vapor team at Nodes met up in Berlin to participate in the conference and with this blog post we wanted to share a couple of highlights from the conference as well as giving some feedback on the conference in case any readers are considering to go next year.
+
+## Quick facts
+Some quick facts about ServerSide.swift 2018:
+- September 12-14 in Berlin (Adlershof)
+- One day of workshops. Four half-day workshops to choose from where two ran in parallel before lunch and the last two ran in parallel after lunch. This meant that you would be able to attend two in total.
```

Comment:
Four half-day workshops to choose from, with two running in parallel before lunch and two more running in parallel after lunch. This meant that you would be able to attend two at most.